hack/tests/e2e-ansible-molecule.sh

```diff
@@ -56,15 +35,21 @@ else
 fi
 KUSTOMIZE_PATH=${KUSTOMIZE} TEST_OPERATOR_NAMESPACE=default molecule test -s kind
 
-popd
-popd
+cd $TMPDIR
 KUSTOMIZE_PATH=${KUSTOMIZE}
 header_text "Test Ansible Molecule scenarios"
 pushd "${ROOTDIR}/test/ansible"
```

Comment:
See that a pr to change it will be done as a follow up: https://github.com/operator-framework/operator-sdk/pull/4011/files

c/c @asmacdo @fabianvf